### DISCUSSION TITLE
How to train a custom ckpt other than the regular v1.4 in dreambooth?

### DISCUSSION THREAD

**Discussion by giteeeeee** [TheLastBen/fast-stable-diffusion#187] [Q&A] [answered]
I have a custom ckpt other than the regular stable-diffusion-v-1-4.ckpt

I want to train it using the dreambooth colab. But where do I put this custom ckpt in colab?

**Comment by TheLastBen** [comment, marked as answer]
Update your notebook to the latest to use the feature that allows to train a custom CKPT
https://github.com/TheLastBen/fast-stable-diffusion/commit/d14a4eeb7552b4fa26beca997290da34d3ee2d8f
